src/client/CourseRequestForm/index.js

```diff
@@ -85,21 +91,21 @@ const CourseRequestForm = ({ courses }) => {
     coursesData[index].toBeBuilt = !coursesData[index].toBeBuilt;
     setCoursesData(coursesData);
   };
-  const date = new Date();
-  // Const updatedDate = `${date.getFullYear()}-${}`;
+
+  const markedDate = moment(new Date()).format('YYYY-MM-DD h:mm A');
```

Comment:
@tatikamsetti Reading up more on Moment.js and it seems the project is now defunct: https://momentjs.com/docs/#/-project-status/

Should be able to get similar results using:

Date.toLocaleDateString() and Date.toLocaleTimeString()

Reply:
@rlorenzo Ok changed it.